src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -3397,7 +3403,10 @@ namespace HVACUnitaryBypassVAV {
         Real64 thisTime = (dayOfSim - 1) * 24 + DataGlobals::HourOfDay - 1 + (DataGlobals::TimeStep - 1) * DataGlobals::TimeStepZone +
                           DataHVACGlobals::SysTimeElapsed;
 
-        if (thisTime <= CBVAV(CBVAVNum).changeOverTimer) return;
+        if (thisTime <= CBVAV(CBVAVNum).changeOverTimer) {
+            CBVAV(CBVAVNum).modeChanged = true;
+            return;
+        }
```

Comment:
Now here, the mode has changed and this function is not needed so a return is used. But this defect showed that the SPMs needed to be called so the modeChanged flag also needs to stay true during the min time that the operating mode must be active (e.g., for at least 0.5 hours)